docs/topics/modules/run_debug.adoc

```diff
@@ -3,22 +3,23 @@
 
 After setting up the Debugger, you can use the Debug command for your project:
 
-. Click btn:[Run] from the menu options.
-. Select the btn:[Debug] option and then the btn:[Remote Java] option.
+. Navigate to menu:Run[Debug>Remote Java] from the menu options.
 +
 image::debug_remote_java.png[Debug Remote Java]
 +
-. A message appears when the Remote debugger connects.
+
+. A message appears when the remote debugger connects.
```

Comment:
Don't use 'appear' (IBM SG). Instead, use 'is displayed', or "<name-of-component> displays a message" to avoid passive.